test/services/ForeignPropertyServiceSpec.scala

```diff
@@ -87,289 +69,33 @@ class ForeignPropertyServiceSpec
-      val periodicSubmissionIds = List(
-        PeriodicSubmissionIdModel(periodicSubmissionId, LocalDate.parse("2023-04-06"), LocalDate.parse("2024-04-05"))
-      )
-      val propertyPeriodicSubmission = PropertyPeriodicSubmission(
-        Some(PeriodicSubmissionId(periodicSubmissionId)),
-        submittedOn = Some(LocalDateTime.now),
-        fromDate = LocalDate.now.minusDays(1),
-        toDate = LocalDate.now,
-        None,
-        None
-      )
-
-      mockGetAllPeriodicSubmission(taxYear, nino, incomeSourceId, Right(periodicSubmissionIds))
-      mockGetPropertyPeriodicSubmission(taxYear, nino, incomeSourceId, "1", Right(Some(propertyPeriodicSubmission)))
-
-      await(underTest.getPropertyPeriodicSubmissions(taxYear, nino, incomeSourceId).value) shouldBe
-        Right(PropertyPeriodicSubmissionResponse(List(propertyPeriodicSubmission)))
-    }
-
-    "return DataNotFoundError when GetPeriodicSubmission has ids and the period is less than a year" in {
-      val aPeriodicSubmissionModel = List(
-        PeriodicSubmissionIdModel("1", LocalDate.parse("2021-01-01"), LocalDate.parse("2021-11-11"))
-      )
-
-      mockGetAllPeriodicSubmission(taxYear, nino, incomeSourceId, Right(aPeriodicSubmissionModel))
-
-      await(underTest.getPropertyPeriodicSubmissions(taxYear, nino, incomeSourceId).value) shouldBe Right(
-        PropertyPeriodicSubmissionResponse(List())
-      )
-    }
-
-    "return DataNotFoundError when GetPeriodicSubmission has ids and there is no submission" in {
-      val aPeriodicSubmissionModel = List(
-        PeriodicSubmissionIdModel("1", LocalDate.parse("2021-01-01"), LocalDate.parse("2021-11-11"))
-      )
-
-      mockGetAllPeriodicSubmission(taxYear, nino, incomeSourceId, Right(aPeriodicSubmissionModel))
-
-      await(underTest.getPropertyPeriodicSubmissions(taxYear, nino, incomeSourceId).value) shouldBe Right(
-        PropertyPeriodicSubmissionResponse(List())
-      )
-    }
-
-    "return DataNotFoundError when GetPeriodicSubmission does not have ids" in {
-      val aPeriodicSubmissionModel = List.empty[PeriodicSubmissionIdModel]
-
-      mockGetAllPeriodicSubmission(taxYear, nino, incomeSourceId, Right(aPeriodicSubmissionModel))
-
-      await(underTest.getPropertyPeriodicSubmissions(taxYear, nino, incomeSourceId).value) shouldBe Right(
-        PropertyPeriodicSubmissionResponse(List())
-      )
-
-    }
-
-    "return ApiError when GetPeriodicSubmissionIds fails" in {
-      mockGetAllPeriodicSubmission(
-        taxYear,
-        nino,
-        incomeSourceId,
-        Left(ApiError(INTERNAL_SERVER_ERROR, SingleErrorBody("code", "error")))
-      )
-      await(underTest.getPropertyPeriodicSubmissions(taxYear, nino, incomeSourceId).value) shouldBe Left(
-        ApiServiceError(500)
-      )
-    }
-  }
-
-  val validCreatePropertyPeriodicSubmissionRequest: CreatePropertyPeriodicSubmissionRequest =
-    CreatePropertyPeriodicSubmissionRequest(
-      LocalDate.now(),
-      LocalDate.now(),
```

Comment:
why have these tests been removed ?

Reply:
Something went wrong in rebase. Reverted. Thanks